ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -116,7 +123,8 @@ message CreatedEvent {
   // Metadata of the contract. Required for contracts created
   // after the introduction of explicit disclosure.
   // Optional
-  ContractMetadata metadata = 10;
+
+  ContractMetadata metadata = 10 [deprecated = true];
```

Comment:
Explain what should be used instead, i.e., "deprecated in favor of contract_create_payload"